homeassistant/components/zha/api.py

```diff
@@ -74,6 +76,38 @@
 }
 
 
+@websocket_api.require_admin
+@websocket_api.async_response
+@websocket_api.websocket_command({
+    vol.Required('type'): 'zha/devices/add'
```

Comment:
Do you think that "Add" to be the right terminology ? Zigbee calls it pairing, and I wonder if that is also what users will expect / will look for based on their manuals?

Reply:
I’m not married to the names. I can change it to zha/devices/pair if you think that’s a better name.

Reply:
You have a point.